fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt

```diff
@@ -612,4 +614,19 @@ class InterceptorTest : BaseTestCase() {
         assertThat(data, nullValue())
         assertThat(error, notNullValue())
     }
+
+    @Test
+    fun testRemoveAllRequestInterceptors() {
+        val manager = FuelManager()
+        val (request, response, _) = manager.request(Method.GET,
+                "http://httpbin.org/status/418").response()
+
+        //val (data, error) = result
```

Comment:
Uncomment?

Reply:
There is a reason that I didn't comment these, those are null.

Reply:
I could remove these comments if you want.

Reply:
Hmm do you know why though? If that's the case, we might not be able to properly test the effect of calling `removeAllRequestInterceptors `